src/main/java/org/verdictdb/core/sqlobject/ConstantColumn.java

```diff
@@ -25,6 +25,8 @@
 
   private static final long serialVersionUID = -4530737413387725261L;
 
+  public enum databaseDataType {intType};
```

Comment:
For MySQL, to cast column into int type, we need to cast unsigned/signed instead of cast int. So I intend to pass a ConstantColumn that contains this value to the create method of cast columnOp to indicate we need to cast it to int type.

Reply:
If that's the case, I think it would be cleaner to let the sql writer handle the conversion. The sqlobject classes are supposed to be database-independent. Concretely, let the database-independent expression be ColumnOp("cast", column instance, "int"). Inside the sql writer, resort to MySqlSyntax (and other syntax classes) to know if any other type names must be used in place of "int".